### PR TITLE
Allow different users to deploy to the same host

### PR DESCRIPTION
**Pull request by karenc**
- :bug: Fix permissions for planted source
 - Build webview on debian without using sudo
 - Install webview as www-data on debian systems
 - Install python packages as www-data on debian systems

### CODE REVIEW

File: tasks/install_arclishing.yml [hunk 1]
```diff
@@ -72,15 +72,28 @@
...
     mode: 0755
-    owner: "{{ ansible_user_id }}"
+    owner: "{{ ansible_os_family == 'Darwin' and ansible_user_id or 'www-data' }}"
+
+- name: set the owner of venvs directory (on Debian)
+  when: ansible_os_family == 'Debian'
+  become: yes
+  file:
+    path: "{{ root_prefix }}/var/cnx/venvs"
+    state: directory
```
Comment (pumazi) [inline tasks/install_arclishing.yml]: I'm wondering if this task should be using 'touch' rather than directory. Or whether it makes a difference.

**Comment by pumazi** [comment]
I see the ``webview/dist`` directory is still assigned to the ansible_user_id. We may need to touch up that task as well.

**Comment by karenc** [comment]
Since `webview/dist` is on the local machine, I think it is ok to leave it assigned to the ansible_user_id.

**Comment by karenc** [comment]
Ok I think I got everything that I want in.  Ready for review.

**Pull request merged**